mergify_engine/actions/merge/action.py

```diff
@@ -50,7 +50,10 @@ class MergeAction(actions.Action):
             "merge", "squash", None
         ),
         voluptuous.Required("strict", default=False): voluptuous.Any(
-            bool, "smart", "smart+ordered"
+            bool,
+            voluptuous.All("smart", voluptuous.Coerce(lambda _: "smart+ordered")),
```

Comment:
👏 that hack
I would have done:
```suggestion
            voluptuous.All("smart+ordered", voluptuous.Coerce(lambda _: "smart")),
```

and keep just `smart` as the canonical name, but as you wish.

Reply:
I prefer to keep the long name for the code readability.